hydromosaic/indexing/index_netCDF.py

```diff
@@ -229,7 +229,7 @@ def index_directory(dsn, directory, log_level, gcm_prefix):
             # database objects derived from nc file data
             outlets = get_outlets(nc, session)
             variables = get_variables(nc, session)
-            datafile = get_datafile(os.path.abspath(f"{directory.strip('/')}/{file}"), session)
+            datafile = get_datafile(os.path.abspath(f"{directory.rstrip('/')}/{file}"), session)
```

Comment:
Caught this one fly by my email. As a suggestion something like

`os.path.join(directory, file)`  might be more appropriate here, and will deal with separators for you. as per [here](https://www.geeksforgeeks.org/python-os-path-join-method/)

Reply:
Thanks!